x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/manage_processor_form/manage_processor_form.container.tsx

```diff
@@ -56,12 +56,7 @@ export const ManageProcessorForm: FunctionComponent<Props> = ({
   useEffect(() => {
     const subscription = form.subscribe(onFormUpdate);
     return subscription.unsubscribe;
-
-    // TODO: Address this issue
-    // For some reason adding `form` object to the dependencies array here is causing an
-    // infinite update loop.
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [onFormUpdate]);
+  }, [onFormUpdate, form]);
```

Comment:
nit: it'd be better to deconstruct `subscribe` and only have that dependency. But at the same time, this will have to be refactored to not use subscribe at some point:

```js
const { isValid, validate } = form;

useEffect(() => {
  onFormUpdate({ isValid, validate });
}, [isValid, validate, onFormUpdate]);
```